crates/fluvio-controlplane-metadata/src/mirror/status.rs

```diff
@@ -36,9 +68,11 @@ pub enum MirrorPairStatus {
     #[fluvio(tag = 1)]
     Succesful,
     #[fluvio(tag = 2)]
-    Failed,
+    Failed(String),
```

Comment:
This will break existing API.  Instead maybe
```
#[fluvio(tag = 2)]
Failed,
#[fluvio(tage=4]
DetailFailure(String)
```

And provide compatibility with previous version.  Need to increment protocol version as well

Reply:
Sure! COMMON_VERSION is the protocol version, right?

So would be something like it?

`pub(crate) const COMMON_VERSION: i16 = 17;`

and

```
#[fluvio(tag = 5, min_version = 17)]
DetailFailure(String),
```